.github/ISSUE_TEMPLATE/instruction-suggestions.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Instruction suggestions
+about: Suggest new instructions for Relaxed SIMD
+title: ''
+labels: instruction-proposal
+assignees: ''
+
+---
+
+1. What are the instructions being proposed?
+
+2. What are the semantics of these instructions?
+
+3. How will these instructions be implemented? Give examples for at least x86-64 and ARM64.
```

Comment:
I suggest we ask contributors to provide a reference implementation expressed through WAsm SIMD128 instructions. This reference implementation would be used by engines which strictly favor portability over performance.